tests/func-tests/observability_controller_test.go

```diff
@@ -55,3 +68,13 @@ var _ = Describe("Observability Controller", Label(tests.OpenshiftLabel, "observ
 		})
 	})
 })
+
+func serviceAccountTlsCertPathExists() bool {
+	_, err := os.ReadFile(observability.ServiceAccountTlsCertPath)
+	if errors.Is(err, os.ErrNotExist) {
+		return false
+	}
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Please avoid using gomega asserion in helper function (in new code...).

Please return the error, and asset it from the caller code; e.g. change the function signature to return bool and error.

Reply:
updated